packaging/build_cmake.sh

```diff
@@ -42,7 +42,14 @@ else
   PYTORCH_MUTEX_CONSTRAINT=''
 fi
 
-conda install -yq \pytorch=$PYTORCH_VERSION $CONDA_CUDATOOLKIT_CONSTRAINT $PYTORCH_MUTEX_CONSTRAINT $MKL_CONSTRAINT -c "pytorch-${UPLOAD_CHANNEL}"
+if [[ "$CU_VERSION" == "cu111" ]]; then
+  # cuda-toolkit=11.1 is only available on conda-forge
+  CONDA_EXTRA_FLAGS="-c conda-forge"
```

Comment:
@fmassa I remember there was a discussion prior my leave on June about using forge, but not sure what was decided?

Reply:
conda-forge is still needed if one wants to install CUDA-11.1, but I will probably move the config to CUDA-11.3 and be done with it.